### PR TITLE
first try to resolve issue #157

### PR DESCRIPTION
**Pull request by schmelto**
first try to resolve issue #157

### CODE REVIEW

File: src/options/options.js [hunk 1]
```diff
@@ -20,3 +20,5 @@ RandomTips.init(tips).then(() => {
     RandomTips.setContext("options");
     RandomTips.showRandomTipIfWanted();
 });
+
+document.getElementById("popupShowContextMenu").checked = true;
```
Comment (rugk) [inline src/options/options.js:24]: Nope, this is not how we do it in this project. See [this answer](https://github.com/rugk/offline-qr-code/issues/157#issuecomment-508475513) and [this doc](https://github.com/TinyWebEx/AddonSettings#default-values) (also consider to read the whole one). You have to define the default setting in `DefaultSettings.js`.

File: src/options/options.html [hunk 2]
```diff
@@ -188,6 +188,13 @@ <h1 data-i18n="__MSG_titleAddonBehavior__">Addon behavior</h1>
 							<span data-i18n="__MSG_optionDebugModeDescr__" class="helper-text">This is only useful to get more detailed logging output in the console for reporting bugs, etc.</span>
 						</div>
 					</li>
+
+					<li>
+						<div class="line">
+							<input class="setting save-on-change" type="checkbox" id="popupShowContextMenu" name="popupShowContextMenu">
+							<label data-i18n="__MSG_optionPopupShowContextMenu__" for="popupIconShowContextMenu">Show context menu item for selected text</label>
```
Comment (rugk) [inline src/options/options.html:195]: All `__MSG` things need localisation. (At least the English one if you speak no other language.) See [this answer again](https://github.com/rugk/offline-qr-code/issues/157#issuecomment-508475513) and [this doc](https://github.com/TinyWebEx/Localizer).

**Review by rugk — CHANGES_REQUESTED** [review]
Hi @schmelto,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

This is a start, but surely does not work yet. Or did you check the button and notice any difference?

Here is what needs to be done:
* the settings need to be made translatable (i.e. localize them, we have a module for this), see comment
* you need to define the default value, see comment
* most importantly: you need to read the settings value somewhere ([see this module](https://github.com/TinyWebEx/AddonSettings#getting-settings)) and then add the logic to actually hide or show the context menu entries.

Also, BTW, it may be a good idea to adjust the title of this PR to explain what you did and not only link to the issue number.
You can also (automatically) [let issues close](https://help.github.com/articles/closing-issues-via-commit-messages/) when a PR is merged by adding some "magic" text to your PR _body_. (Manually linked it now.)

BTW: Next time, try to avoid doing a pull request from the master branch, because you can run into problems when you have a "non-clean" master that does not follow this repo here (i.e. "upstream"). [See this article for details](https://help.github.com/en/articles/proposing-changes-to-your-work-with-pull-requests). Anyway, this is only a tip for the next time. :smiley:

**Comment by rugk** [comment]
@schmelto  Are you still interested in completing this? If not, that's no problem, just add a notice, so I know what the current status of this PR is.

**Comment by schmelto** [comment]
@rugk sorry I'll give it a try at this weekend

**Comment by rugk** [comment]
@schmelto  Are you still interested in completing this? If not, that's no problem, just add a notice, so I know what the current status of this PR is.

**Comment by schmelto** [comment]
Hey @rugk think this is outdated from me and someone else can pick it up :)

**Pull request closed without merging**